include/libopencm3/ethernet/mac_stm32fxx7.h

```diff
@@ -91,8 +90,8 @@
 #define ETH_PTPTSAR			MMIO32(ETHERNET_BASE + 0x718)
 #define ETH_PTPTTHR			MMIO32(ETHERNET_BASE + 0x71C)
 #define ETH_PTPTTLR			MMIO32(ETHERNET_BASE + 0x720)
-/* not available on F1 ?*/
-#define ETH_PTPTSSR			MMIO32(ETHERNET_BASE + 0x728)
+#define ETH_PTPTSSR			MMIO32(ETHERNET_BASE + 0x728) /* Not on STM32F1 */
+#define ETH_PTPPPSCR		MMIO32(ETHERNET_BASE + 0x72C) /* Not on STM32F1 */
```

Comment:
heh, I had fun valdating this, in my revision of the f7 ref man, it's in the regster section, but not the register _map_ section!